packages/position-managers/src/types.ts

```diff
@@ -17,6 +17,7 @@ export enum Strategy {
   PASSIVE,
   PEGGED,
   ALO,
+  ASCEND,
```

Comment:
we need to have new strategy name as well?

Reply:
yeah this vault is exception. if u read messages in chat you will see that theres new strategy for this one